react/features/notifications/components/Notification.web.js

```diff
@@ -79,6 +74,12 @@ class Notification extends Component {
         descriptionKey: PropTypes.string,
 
         /**
+         * Whether the support link should be hidden in the case of an error
+         * message.
+         */
+        hideErrorSupportLink: PropTypes.bool,
```

Comment:
Nit: maybe the word error shouldn't be included? I'm not 100% sure of its usage but could the link also be shown in non-error messages?

Reply:
No, the link is _only_ shown in error messages and that's why I need to add some word about it

Reply:
Can't think of a better name.. but hideSupportLink doesn't fit because it only shows for errors.